src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -59,11 +59,6 @@ public CoreNode(NodeRunner runner, NodeBuilder builder, string configfile)
             this.ConfigParameters.Import(builder.ConfigParameters);
             this.ports = new int[3];
             IpHelper.FindPorts(this.ports);
-
-            var loggerFactory = new ExtendedLoggerFactory();
-            loggerFactory.AddConsoleWithFilters();
-
-            this.networkPeerFactory = new NetworkPeerFactory(network, DateTimeProvider.Default, loggerFactory, new PayloadProvider().DiscoverPayloads(), new SelfEndpointTracker());
```

Comment:
Is this change related to this PR?

Reply:
I think it's a merge thing that was left behind.